src/native/http_connection_manager.h

```diff
@@ -6,6 +6,7 @@
 #ifndef AWS_JNI_CRT_HTTP_CONNECTION_MANAGER_H
 #define AWS_JNI_CRT_HTTP_CONNECTION_MANAGER_H
 
+#include <http_proxy_options.h>
```

Comment:
trivial: move this include to `http_connection_manager.c`, it doesn't need to be in the header file.